styles/styles.css

```diff
@@ -168,6 +168,11 @@ h4 { font-size: var(--heading-font-size-m) }
 h5 { font-size: var(--heading-font-size-s) }
 h6 { font-size: var(--heading-nav-font-size) }
 
+.blogpost h2 {
+  font-size: 24px;
+  color: var(--c-dark-plum);
```

Comment:
This can be removed also mostly all other headers are `--c-dark-plum` should not be specific for the blog